concourse/tasks/delete_admin.yml

```diff
@@ -31,3 +32,6 @@ run:
       echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
 
       cf delete-user "${NAME}" -f
+
+      echo "Removing leaked users"
+      cf curl /v2/users | jq -r ".resources[] | .entity.username | select(. != null) | select(test(\"^${PREFIX}\"))" | xargs -r -I {} cf delete-user {} -f
```

Comment:
Thanks, it wasn't clear there was a default for `results-per-page` from the API documentation, and my development environment was too small to hit this, so I assumed unpaged by default.

I'll look at doing the pagination as I don't think it'll be very complex.